infra/feast-operator/internal/controller/services/util.go

```diff
@@ -182,6 +176,24 @@ func ensureRequestedStorage(resources *v1.VolumeResourceRequirements, requestedS
 	}
 }
 
+func ensurePVCDefaults(pvc *feastdevv1alpha1.PvcConfig, feastType FeastServiceType) {
+	var storageRequest string
+	switch feastType {
+	case OnlineFeastType:
+		storageRequest = DefaultOnlineStorageRequest
+	case OfflineFeastType:
+		storageRequest = DefaultOfflineStorageRequest
+	case RegistryFeastType:
+		storageRequest = DefaultRegistryStorageRequest
+	}
```

Comment:
maybe `storageRequest` could be an input instead?
```suggestion
func ensurePVCDefaults(pvc *feastdevv1alpha1.PvcConfig, feastType FeastServiceType, storageRequest string) {
```